examples/java_postgresql_customvar/README.md

```diff
@@ -237,12 +350,43 @@ spec:
               name: binding-request
 ```
 
-#### ServiceBindingRequestStatus
+### Check the environment variable creation in the application pod
 
-`ServiceBindingRequestStatus` depicts the status of the Service Binding operator. More info: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#spec-and-status
+To list all the pods, run the command:
+
+```
+➜  ~ oc get pods                               
+NAME                                   READY   STATUS      RESTARTS   AGE
+db-demo-postgresql-78b8466897-ht8x9    1/1     Running     0          15m
+java-rest-http-crud-1-build            0/1     Completed   0          51m
+java-rest-http-crud-666b8597cc-w7456   1/1     Running     0          15m
+```
+
+To ssh into the application pod `java-rest-http-crud-666b8597cc-w7456`, run command:
 
-| Field | Description |
-|-------|-------------|
-| Secret | The name of the intermediate secret |
+```
+oc exec -it java-rest-http-crud-666b8597cc-w7456 /bin/bash
+```
+
+Print all the environment variables that were injected into the application pod by Service Binding Operator:
+
+```
+[jboss@java-rest-http-crud-666b8597cc-w7456 ~]$ printenv | grep DATABASE_
```

Comment:
@isutton Why are the environment variables appended with `DATABASE_` prefix
because I haven't mentioned any prefix (global or service-level) in the SBR yaml.
Is this expected behavior?

Reply:
I understand this now. The variable prefix is actually `Kind` of backing Service.

Reply:
I think by default it takes the `Kind` of the GVK mentioned as a selector